pymc3/tests/test_plots.py

```diff
@@ -0,0 +1,54 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+from arviz import from_pymc3
+
+import pymc3 as pm
+
+from pymc3.backends.ndarray import point_list_to_multitrace
+from pymc3.plots import plot_posterior_predictive_glm
```

Comment:
This doesn't seem `isort`ed

Reply:
isort sorts as follows:

- standard library
- 3rd party libraries
- 1st party libraries

We're then imposing that `import x` be separated from `from y import x` in each section. This looks a bit different to how the imports are sorted in the notebooks, because there `pymc3` was treated as a third party, while here it's a first party. So...we can either accept these defaults, or in `setup.cfg` we can impose that even in notebooks `pymc3` be treated as a first party, or here we group `pymc3` with 3rd parties

Reply:
Ah ok! No it's alright, let's keep it like this